### DISCUSSION TITLE
Projects or Repositories

### DISCUSSION THREAD

**Discussion by PedroDRodrigues** [community/community#43618] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi! I am new to GitHub and the development environment, so that's why I am here to ask what is the best way to share my personal projects. I am not sure if it is better to share the projects in the general topic or in repositories topic. Can you share your opinion?

Really thank you!

**Comment by maaicondgl** [comment]
acho que é melhor compartilhar no topico de repositorios, fica mais organizado

**Comment by ghost** [comment]
Olá, Pedro Dias Rodrigues, penso que a melhor forma de partilhar os seus projetos pessoais é através de promoções nas suas redes sociais, ou afiliações com outros utilizadores da plataforma Github, para incentivar a atividade e desenvolvimento dos seus repositórios.
